Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Body-Mass Index (BMI) Calculator");
+            Console.WriteLine("");
+
+            ProcessPersonBMI();
+            ProcessPersonBMI();
+        }
+
+        private static void ProcessPersonBMI()
```

Comment:
Try to keep it conventional and don't have more than 1 capitalized letter in a row. Use PascalCase.
``ProcessPersonBMI`` should be ``ProcessPersonBmi``